tests/end_to_end/function-pointer2/irix-o2-out.c

```diff
@@ -1,5 +1,5 @@
-s32 bar(f32); // extern
-extern s32 (*glob2)(f32);
```

Comment:
(it'd be nice to avoid parameter names in types. but that's pre-existing ofc)

Reply:
This is actually an easy fix, I could do it after this PR? (it'll have a big diff)